scripts/common/_common.sh

```diff
@@ -30,6 +30,7 @@ export HOST_DIR=$OUTPUT_ROOT/corehost
 [ -z "$DOTNET_HOME" ] && export DOTNET_HOME=$STAGE2_DIR && export PATH=$STAGE2_DIR/bin:$PATH
 [ -z "$CONFIGURATION" ] && export CONFIGURATION=Debug
 [ -z "$NOCACHE" ] && export NOCACHE=""
+[ -z "$DOTNET_DEB_PACKAGE_NAME" ] && export DOTNET_DEB_PACKAGE_NAME="dotnet-nightly"
```

Comment:
not sure the 'reasonable default' for this export belongs here. It's pretty specific to the debian packaging domain...